src/signframework/sign_cert/tests/issuer_final_sample_aux.cfg

```diff
@@ -0,0 +1,25 @@
+# root CA auxiliary configuration file
+
+sign_algorithm=RSA-2048
+digest_algorithm=SHA-256
+
+
+# certificate validity in years
+validity=20
```

Comment:
Maybe pick a more "standard" validity range. I think the "standard" right now is a max of 2 years?